templates/allocation/allocation_list.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
+
+
+{% block content %}
+    <div class="p-2 w-50 mx-auto">
+        <h2>Allocation Settings</h2>
+        <a class="mb-4" href="/admin/consvc_shepherd/allocationsetting/add/">Add New Position</a>
+        {% for a_s in allocation_settings %}
```

Comment:
Wondering if it might be nice to have a clear indicator that no settings are defined? In that case, just having a check on `allocation_settings` to render a message that clearly states no allocation is present? Not sure if it's a valuable use case, but an easy addition for clarity. What do you think?

```suggestion
        <a class="mb-4" href="/admin/consvc_shepherd/allocationsetting/add/">Add New Position</a>
    {% if not allocation_settings %}
    <h5>No allocations set.</h5>
    {% endif %}
        {% for a_s in allocation_settings %}
```